src/main/java/org/opensearch/eval/Constants.java

```diff
@@ -61,11 +97,36 @@ public class Constants {
             "          }";
 
     /**
-     * The metrics index mapping.
+     * THe name of the index that stores the results from each query in a query set run.
+     */
+    public static final String QUERY_RESULTS_INDEX_NAME = "srw_query_results";
+
+    /**
+     * The query sets index mapping.
+     */
+    public final static String QUERY_RESULTS_MAPPING = "{\n" +
+            "              \"properties\": {\n" +
+            "                \"id\": { \"type\": \"keyword\" },\n" +
+            "                \"timestamp\": { \"type\": \"date\", \"format\": \"strict_date_time\" },\n" +
+            "                \"query_set_id\": { \"type\": \"keyword\" },\n" +
+            "                \"query\": { \"type\": \"keyword\" },\n" +
```

Comment:
Haven't noticed that before: we have an inconsistency for storing the user query. It's not consistently stored in the field `user_query` but I think it should so we are aligned with how the UBI specification handles user queries.

I created #124 for this. We can make this alignment in a separate branch or in this one.